health/micro-ui/web/micro-ui-internals/packages/modules/health-payments/src/components/MyBillsTable.js

```
@@ -0,0 +1,230 @@
+    try {
+        const response = await fetch(link, {
+            responseType: "arraybuffer",
+            headers: {
+                "Content-Type": "application/json",
+                Accept: "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet", // MIME type for Excel
+            },
+            method: "GET",
+            mode: "cors",
+        }).then((res) => res.blob());
+
+        if (window.mSewaApp && window.mSewaApp.isMsewaApp() && window.mSewaApp.downloadBase64File) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)